.coafile

```diff
@@ -0,0 +1,27 @@
+[all]
+ignore = node_modules/**, bower_components/**, dist/**, .eslintrc.js, tmp/**,
+         .git/**
+
+[all.SpaceConsistencyBear]
+files = **.js, **.json, **.yml, **.md, .eslintrc
```

Comment:
```console
Executing section all.SpaceConsistencyBear...
[WARNING][21:16:59] No files matching '/home/travis/build/coala/git-task-list/.eslintrc' were found.
```

Reply:
Will remove, and ignore eslint on js. it is invalid name